quickstarts/dapr/config.yml

```diff
@@ -0,0 +1,27 @@
+id: 658d36f3-aa96-4e74-93da-b74eefcaabd3
+slug: dapr
+title: Dapr
+description: |
+  The Dapr quickstart allows you to add meaningful dashboards to New Relic that highlight key aspects of the Dapr platform.
+
+  A guide on how to set-up New Relic for distributed tracing with Dapr can be found [here](https://docs.dapr.io/operations/monitoring/tracing/newrelic/).
+summary: |
+  Dapr is a portable, event-driven runtime that makes it easy for any developer to build resilient, stateless and stateful applications that run on the cloud and edge and embraces the diversity of languages and developer frameworks.
+level: Community
+authors:
+  - Harry Kimpel
+keywords:
+  - apm
+  - dapr
+dashboards:
+  - dapr
+installPlans:
+  - third-party-daprr
+dataSourceIds:
+  - dapr
```

Comment:
Looks like this data source doesn't exist yet, we'll need to create a new `dapr` data source in this PR as mentioned above.

Reply:
added the data source